vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ExhibitionAboutPage.java

```diff
@@ -32,6 +37,16 @@ public class ExhibitionAboutPage {
     
     @Lob
     private String aboutPageText;
+    
+    
+    
+    @OneToMany(targetEntity = LocalizedText.class, cascade={CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH})
+    @JoinTable(name="AboutPage_LocText_titles")
+    private List<ILocalizedText> exhibitionTitles = new ArrayList<ILocalizedText>();
```

Comment:
why is this property called "exhibitionTitles"? It's not really multiple titles for the exhibition. why not `localizedTitles`?